src/client/gui/lib/settings/virtualization_settings.dart

```diff
@@ -42,22 +45,10 @@ class VirtualizationSettings extends ConsumerWidget {
           value: networks.contains(bridgedNetwork) ? bridgedNetwork : null,
           items: Map.fromIterable(networks),
           onChanged: (value) {
-            ref.read(bridgedNetworkProvider.notifier).set(value!);
+            ref.read(bridgedNetworkProvider.notifier).set(value!).onError(
```

Comment:
On Windows (Hyper-V), in settings, the virtual interface setting does not show up right away. After a few seconds, and constantly scrolling down, it shows up. This happens every time I open the settings page.

Reply:
I think this is caused simply by the fact that it takes a bit for the `networks` rpc to respond. The scrolling should have nothing to do with it.
It would be nice if we had some design for how to display a loading operation here. But it might be a little more complicated than that. The only way to know if the `networks` feature is enabled or not on the current backend is to try to call it and see if it responds successfully or not. So if we knew ahead of time if the feature is supported, we could ask for the networks and show a loading indicator in the meantime, or if the feature is not supported, to not even try the request and show nothing at all. But this would require a very useful feature i.e. an rpc for daemon capabilities.